ios/FluentUI/TwoLineTitleView/TwoLineTitleView.swift

```diff
@@ -265,22 +275,26 @@ open class TwoLineTitleView: UIView {
     ///
     /// - Parameters:
     ///   - title: A title string.
+    ///   - titleImage: An optional image to display before the title.
     ///   - subtitle: An optional subtitle string. If nil, title will take up entire frame.
     ///   - alignment: How to align the title and subtitle. Ignored if `subtitle` is nil.
     ///   - interactivePart: Determines which line, if any, of the view will have interactive button behavior.
     ///   - animatesWhenPressed: If true, the text color will flash when pressed. Ignored if `interactivePart` is `.none`.
     ///   - accessoryType: Determines which accessory will be shown with the `interactivePart` of the view, if any. Ignored if `interactivePart` is `.none`.
-    @objc open func setup(title: String, subtitle: String? = nil, alignment: Alignment = .center, interactivePart: InteractivePart = .none, animatesWhenPressed: Bool = true, accessoryType: AccessoryType = .none) {
+    @objc open func setup(title: String, titleImage: UIImage? = nil, subtitle: String? = nil, alignment: Alignment = .center, interactivePart: InteractivePart = .none, animatesWhenPressed: Bool = true, accessoryType: AccessoryType = .none) {
```

Comment:
we determine this api change is fine because client team won't be impacted?

Reply:
Yes, this should be fine. We're already adding a bunch of new arguments here, which is totally fine in Swift since they're optional, but we're also keeping the old signature around for those who are using Objective-C.